src/core/scene/a-scene.js

```diff
@@ -50,10 +50,16 @@ module.exports.AScene = registerElement('a-scene', {
         this.isMobile = isMobile;
         this.isScene = true;
         this.object3D = new THREE.Scene();
+        this.vrCamera = null;
+        var self = this;
+        this.object3D.onAfterRender = function (renderer, scene, camera) {
+          self.vrCamera = camera;
```

Comment:
What is this variable for!

Reply:
the issue with three's internal vr handling was that it was swapping the provided camera internally, with an ArrayCamera for the twin views in the case of VR. This camera was not exposed but needed for post processing and tiled forward rendering. onBeforeRender and onAfterRender where added to address that, the correct camera will be provided in every case.

So .vrCamera will be the same as .camera in the case of non-vr rendering and the ArrayCamera otherwise so it can be used in view dependent post proc filters like god rays.

The naming was completely arbirtrary, if you prefer something else just say so

Reply:
I see. I comment about what the variable is for will be hepful. We can also think about a better name: `cameraUsedToRender` , `rendererInternalCamera` ... not sure. Something more descriptive

Reply:
renderCamera?